pkg/node/node.go

```diff
@@ -744,6 +744,8 @@ func NewBee(interrupt chan struct{}, sysInterrupt chan os.Signal, addr string, p
 	hive.SetAddPeersHandler(kad.AddPeers)
 	p2ps.SetPickyNotifier(kad)
 
+	var statusStampsSynced bool
```

Comment:
we do not need this var, we can move the `post.HandleStamps()` after line 774 and again after 783 (after the error checks)